utils/check-style/check-style

```diff
@@ -16,404 +16,409 @@ LC_ALL="en_US.UTF-8"
-if [[ "$(grep -Po "M\([0-9]*," $ROOT_PATH/src/Common/ErrorCodes.cpp | wc -l)" != "$(grep -Po "M\([0-9]*," $ROOT_PATH/src/Common/ErrorCodes.cpp | sort | uniq | wc -l)" ]]
-then
-    echo "ErrorCodes.cpp contains non-unique error codes"
-fi
-
-# Check that there is no system-wide libraries/headers in use.
+# # From [1]:
+# #     But since array_to_string_internal() in array.c still loops over array
+# #     elements and concatenates them into a string, it's probably not more
+# #     efficient than the looping solutions proposed, but it's more readable.
+# #
+# #  [1]: https://stackoverflow.com/a/15394738/328260
+# function in_array()
+# {
+#     local IFS="|"
+#     local value=$1 && shift
 #
-# NOTE: it is better to override find_path/find_library in cmake, but right now
-# it is not possible, see [1] for the reference.
+#     [[ "${IFS}${*}${IFS}" =~ "${IFS}${value}${IFS}" ]]
+# }
 #
-#   [1]: git grep --recurse-submodules -e find_library -e find_path contrib
-if git grep -e find_path -e find_library -- :**CMakeLists.txt; then
-    echo "There is find_path/find_library usage. ClickHouse should use everything bundled. Consider adding one more contrib module."
-fi
-
-# Forbid files that differ only by character case
-find $ROOT_PATH | sort -f | uniq -i -c | awk '{ if ($1 > 1) print }'
-
-# Forbid std::filesystem::is_symlink and std::filesystem::read_symlink, because it's easy to use them incorrectly
-find $ROOT_PATH/{src,programs,utils} -name '*.h' -or -name '*.cpp' |
-    grep -vP $EXCLUDE_DIRS |
-    xargs grep -P '::(is|read)_symlink' | grep -v "STYLE_CHECK_ALLOW_STD_FS_SYMLINK" && echo "Use DB::FS::isSymlink and DB::FS::readSymlink instead"
-
-# Forbid __builtin_unreachable(), because it's hard to debug when it becomes reachable
-find $ROOT_PATH/{src,programs,utils} -name '*.h' -or -name '*.cpp' |
-    grep -vP $EXCLUDE_DIRS |
-    xargs grep -P '__builtin_unreachable' && echo "Use UNREACHABLE() from defines.h instead"
+# find $ROOT_PATH/{src,base,programs,utils} -name '*.h' -or -name '*.cpp' 2>/dev/null |
+#     grep -vP $EXCLUDE_DIRS |
+#     xargs grep $@ -P '((class|struct|namespace|enum|if|for|while|else|throw|switch).*|\)(\s*const)?(\s*override)?\s*)\{$|\s$|^ {1,3}[^\* ]\S|\t|^\s*(if|else if|if constexpr|else if constexpr|for|while|catch|switch)\(|\( [^\s\\]|\S \)' |
+# # a curly brace not in a new line, but not for the case of C++11 init or agg. initialization | trailing whitespace | number of ws not a multiple of 4, but not in the case of comment continuation | missing whitespace after for/if/while... before opening brace | whitespaces inside braces
+#     grep -v -P '(//|:\s+\*|\$\(\()| \)"'
+# # single-line comment | continuation of a multiline comment | a typical piece of embedded shell code | something like ending of raw string literal
+#
+# # Tabs
+# find $ROOT_PATH/{src,base,programs,utils} -name '*.h' -or -name '*.cpp' 2>/dev/null |
+#     grep -vP $EXCLUDE_DIRS |
+#     xargs grep $@ -F $'\t'
+#
```

Comment:
I see all other code is commented out - it was probably committed by mistake.

Reply:
Ah, I missed that, sorry. Fixed.